tools/build.py

```diff
@@ -275,6 +275,8 @@ def build_libuv():
         # libuv is using gyp. run the system according to build target.
         if opt_target_arch() == 'arm' and opt_target_os() =='nuttx':
             check_run_cmd('./nuttx-configure', [opt_nuttx_home()])
+        elif opt_target_arch() == 'arm' and opt_target_os() =='linux':
+            check_run_cmd('./armlinux-configure')
```

Comment:
Ah, ok I see this is for uv configuration. so the file should have been in the uv source tree. That good.